src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with logical operators (&&, ||) or ternary operator (?:)
-  // without nesting
+  let filteredMen = [];
+
+  century === undefined
+    ? filteredMen = people.filter(elem => elem.sex === 'm')
+    : filteredMen = people
+      .filter(elem => elem.sex === 'm'
```

Comment:
Your code looks intuitive and readable, but you iterate over array twice, can you do it once only?